pkg/services/grafana-apiserver/endpoints/request/request_test.go

```diff
@@ -126,3 +136,105 @@ func TestParseNamespace(t *testing.T) {
 		})
 	}
 }
+
+func TestOutputMediaType(t *testing.T) {
+	tests := []struct {
+		name     string
+		ctx      context.Context
+		req      *http.Request
+		expected *schema.GroupVersionKind
+		ok       bool
+	}{
+		{
+			name: "request for table conversion",
+			ctx:  context.Background(),
+			req: &http.Request{
+				Header: http.Header{
+					"Accept": []string{"application/json;as=Table;g=meta.k8s.io;v=v1"},
+				},
+			},
+			expected: &schema.GroupVersionKind{
+				Group:   "meta.k8s.io",
+				Version: "v1",
+				Kind:    "Table",
+			},
+			ok: true,
+		},
+		{
+			name: "application/json",
+			req: &http.Request{
+				Header: http.Header{
+					"Accept": []string{"application/bson"},
```

Comment:
Should this be json? (if not maybe update the test name or add a comment to clarify it's supposed to mismatch)

Reply:
good catch